account_invoice_tax_required/models/account_invoice.py

```diff
@@ -27,8 +27,8 @@ def _test_invoice_line_tax(self):
 
     @api.multi
     def invoice_validate(self):
-        if not (config['test_enable'] and
-                not self.env.context.get('test_tax_required')):
+        if self.env.context.get('test_tax_required') or not (
+                config['test_enable'] or
+                config["init"].get("l10n_generic_coa")):
```

Comment:
In fact, any l10n_* installation

Reply:
Yikes, really?

Then I'll have to update this